metadata-ingestion/src/datahub/ingestion/source/state/checkpoint.py

```diff
@@ -220,7 +208,7 @@ def to_checkpoint_aspect(
                 pipelineName=self.pipeline_name,
                 platformInstanceId=self.platform_instance_id,
                 runId=self.run_id,
-                config=self.config.json(),
+                config="",
```

Comment:
Can we just remove this??

Reply:
(i guess no due to it being a breaking change?)

Reply:
We can't remove it because the field is not optional right now